### PR TITLE
Add doc Proxy translation in English

### PR DESCRIPTION
**Pull request by wcsoft**
Add doc Proxy translation in English

### CODE REVIEW

File: docs/blog/content/material/proxy.en.md [hunk 1]
```diff
@@ -1,7 +1,295 @@
...
+connections=((core_count*2)+effective_spindle_count)
+```
+
+connection count = ((cores*2)+ count disk).  60 connections is enough at a 32 core machine. so there is no need to set hundreds of connections for Proxy, it's not only wastes resources, but also slows down the speed. 
+
+#### 3.  Optimization of resultset conflation
+
+Proxy communicates with real database by JDBC at present, asynchronous access mode of Netty+MySQLProtocol will be released soon, They will coexist, which way to choose depends on client.
+
+JDBC's resultset in Proxy can cause great memory pressure.  Proxy frontend links to m clients,  when backend transfers data to client in the front, these data will store into Proxy memory. If those data stays long time, there will be no memory left and service will be unavailable in the end. Resultset memory efficiency can be optimized in two ways: one way is to reduce data residence time in proxy; the other is current limiting.
```
Comment (menghaoranss) [inline docs/blog/content/material/proxy.en.md:160]: Missing `后端又对应n个真实数据库`

File: docs/blog/content/material/proxy.en.md [hunk 2]
```diff
@@ -1,7 +1,295 @@
...
+
+ChannelOutboundBuffer of Netty swallows all the data from MySQL when SO_SNDBUF is up to top value, because it is unbounded by default.
+
+SO_RCVBUF of Proxy has free memory due to Netty consumption, it causes MySQL sending data all the time, Netty always fills data into ChannelOutboundBuffer where to lead to serious results - no memory left.
+
+When making it clear, our purpose is that Proxy will not accept MySQL data if Client is blocked.
+
+Netty controls writting cache through WRITE_BUFFER_WATER_MARK param: 
+
+*   when Buffer size is up to high watermark, Netty will not produce data any more, unless it's under low watermark；
```
Comment (menghaoranss) [inline docs/blog/content/material/proxy.en.md:221]: `；` Chinese characters

File: docs/blog/content/material/proxy.en.md [hunk 3]
```diff
@@ -1,7 +1,295 @@
...
+    
+*   Proxy implies MySQL not to send data if SO_RCVBUF is filled with after ChannelOutboundBuffer run out.
+    
+
+So the key of the issue is its value for ChannelOutboundBuffer high watermark.
```
Comment (menghaoranss) [inline docs/blog/content/material/proxy.en.md:226]: Missing `所以，在这种情况下，Proxy所消耗的内存只是ChannelOutboundBuffer高水位线的大小。`

File: docs/blog/content/material/proxy.en.md [hunk 4]
```diff
@@ -1,7 +1,295 @@
...
+
+In order to resolve the problem above, CONNECTION_STRICTLY comes out, it doesn't make use of ResultSet flow which causes memory increasing. CONNECTION_STRICTLY doesn't need to keep in touch with database, it will release connection after retrieving all the data in ResultSet.
+
+We use the same setting max_user_connections=80 as example, the database is routed to 100 tables. Proxy create 80 connection for quering data at first, the other 20 connections will be cached into pool. These 20 connections will be created successfully one after another within quering completion before.
+
+If you feel confused, please keep it in mind CONNECTION_STRICTLY is a scenario that max_user_connections is less than maximums of tables routed to.
+
+### 03 Summary
+Sharding-Sphere has been continuous improvement and development since 2016. A growing number of componies and individuals adaop it, and they provide many successful cases for us.  We will move forward to impove current features, achieve soft transaction, data management and so on in succession. If someone has good ideas or wants to do proposals, welcome to join Sharding-Sphere open source project.
```
Comment (menghaoranss) [inline docs/blog/content/material/proxy.en.md:249]: `data management` - > `data governance`

File: docs/blog/content/material/proxy.en.md [hunk 5]
```diff
@@ -1,7 +1,295 @@
...
+*   https://gitee.com/sharding-sphere/sharding-sphere/
+    
+
+  
+
+  
+
+### Q&A
+
+Q1: What's Sidecar？
```
Comment (menghaoranss) [inline docs/blog/content/material/proxy.en.md:262]: `？`  Chinese characters

File: docs/blog/content/material/proxy.en.md [hunk 6]
```diff
@@ -1,7 +1,295 @@
...
+### Link of Live playback
+
+https://m.qlchat.com/topic/details?topicId=2000001395952730&minimal=1
+
+Do you want to know more about Sharding-Sphere？
+
+**Come to "2018 DAMSChina Data Asset Management Summit"**
+
+**Attend the analysis of lecturer Zhang Liang who is responsible for database development of JD Finance**
+
```
Comment (menghaoranss) [inline docs/blog/content/material/proxy.en.md:295]: May be missing the last picture.

**Comment by menghaoranss** [comment]
In comparsion_en.png， `Performance` should be `Performance loss`.

File: docs/blog/content/material/proxy.en.md (hunk 6 above)
Comment (wcsoft) [reply docs/blog/content/material/proxy.en.md:295]: After Communicating with Trista, there is no need to add the picture ~.

**Comment by wcsoft** [comment]
Correct all of translations as metions above, pls review.
https://github.com/apache/shardingsphere/pull/6330

**Pull request closed without merging**